users/yanfali/rules.mk

```
@@ -1,24 +1,29 @@
 BOOTMAGIC = lite
```

Shouldn't this be:
```suggestion
BOOTMAGIC_ENABLE = lite
```
?

---

good catch